.github/workflows/ci.yml

```
@@ -5,16 +5,38 @@ on:
       - master
   pull_request:
 jobs:
+  ubuntuUnitWithJDK8And11:
+    name: ubuntu-latest tests
```

can you include the jdk version in the name?

```suggestion
    name: ubuntu-latest tests (${{  matrix.java-version }})
```

or something like this

---

yep